pontoon/projects/views.py

```diff
@@ -102,7 +94,7 @@ def ajax_notifications(request, slug):
         # For performance reasons, only filter contributors for selected
         # locales if different from all project locales
         available_ids = sorted(list(available_locales.values_list('id', flat=True)))
-        selected_ids = sorted(map(int, selected_ids.split(',')))
+        selected_ids = sorted(map(int, form.cleaned_data.get('selected_locales').split(',')))
```

Comment:
That's deprecated, so I added `validate_comma_separated_integer_list`.

I'm not sure we can simplify getting data from the field, though.